astronomer/airflow/version_check/update_checks.py

```diff
@@ -230,7 +231,7 @@ def parse_version(rel):
             ver = version.parse(release['version'])
             if release['channel'] in ['alpha', 'beta']:  # ignore alpha & beta releases
                 continue
-            if ver <= current_version:
+            if ver < current_version:
```

Comment:
hmm, can't we stop when we see our current version? The debug message is misleading, but this check is right no?

Reply:
Yes, I have changed the debugging message but this condition should be there as we should have current version as well in the database.

Reply:
So shouldn't we switch back to `<=`?

Reply:
No, we need the current version as well in the database to check for EOL. Currently code in main only populates version greater than current version in database.

Reply:
You are right. Sorry ignore the above message. I have created another method to add current runtime version

Reply:
I have also switched back to `<=`

Reply:
As per this conversation it should be `<`. https://github.com/astronomer/astronomer-airflow-version-check/pull/72#discussion_r1655503550

Reply:
Oh, we should update the debug log then?

Reply:
Yes, the log is now wrong.